deployments/jwebbinar/image/Dockerfile.custom

```diff
@@ -57,6 +57,11 @@ RUN   /opt/common-scripts/install-common  base
 
 # --------------------------- Jwebbinar -------------------------------------
 
+# Download Imviz notebook; remove after May 31 2022
+#RUN wget -O /home/jovyan/ImvizExample.ipynb https://raw.githubusercontent.com/spacetelescope/jdaviz/main/notebooks/ImvizExample.ipynb
+#COPY ImvizExample.ipynb /home/jovyan/ImvizExample.ipynb
+ADD https://raw.githubusercontent.com/spacetelescope/jdaviz/main/notebooks/ImvizExample.ipynb /home/jovyan/
```

Comment:
Based on the comment in line 60,  should the ADD be there?

Reply:
I should have updated that comment.  ADD turned out to be the better way.

Reply:
Sorry,  I missed/forgot about your opening comment on the PR which mentioned this,  dumb question.

One thing did occur to me though:  does this fetch need to be in the post-startup-hook so it winds up in EFS instead of the image's  $HOME?